vizro-ai/docs/pages/user-guides/add-generated-chart-usecase.md

```diff
@@ -103,4 +105,55 @@ This object must then be passed to the `figure` argument of the Vizro [Graph](ht
     [VizroDashboard]: ../../assets/user_guides/chart_into_dashboard_large.gif
 
 
+### Direct use of `fig` object
+
+Additionally, we can assign the output of Vizro-AI `plot()` method directly to the fig variable, enabling its reuse in the `vm.Graph.figure` argument.
+This method offers streamlined efficiency, eliminating the need for code copying.
+However, it is important to note that each dashboard run triggers an API call to OpenAI, potentially escalating costs, but this only occurs if the fig object is not stored and is re-run by a Vizro-AI `plot()` call.
```

Comment:
```suggestion
However, it is important to note that each dashboard run triggers an API call to OpenAI, potentially escalating costs, although this can be avoided if the `fig` object is stored and retrieved as needed, instead of making repeated calls to `plot()`.
```